x-pack/plugins/lens/server/migrations/types.ts

```diff
@@ -258,3 +258,16 @@ export interface VisState820 {
   rowHeight: 'auto' | 'single' | 'custom';
   rowHeightLines: number;
 }
+
+export type LensDocShape830<VisualizationState = unknown> = LensDocShape810<VisualizationState>;
+
+export interface XYVisualizationStatePre830 extends VisState820 {
+  valueLabels: 'hide' | 'inside' | 'outside';
+}
+
+export interface XYVisualizationState830 extends VisState820 {
+  valueLabels: ValueLabelConfig;
+}
+
+export type VisStatePre830 = XYVisualizationStatePre830 & VisState820;
```

Comment:
maybe there's something I don't understand about typescript, but given that `XYVisualizationStatePre830` already extends `VisState820` isn't this redundant and basically is the same as `export type VisStatePre830 = XYVisualizationStatePre830`?
Here's the `Bear` example that I understand this way 🤔  https://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases

Reply:
Oh, that totally makes sense. I've missed, that already extended from the VisState820 )

Reply:
Fixed.